src/widgets/WidgetMainFarmers.vue

```diff
@@ -6,11 +6,8 @@
   >
     <template #logo>
       <CardsLogo>
-        <Card>
-          <img src="../assets/images/humm90Card.png" alt="Humm90 card" />
-        </Card>
-        <Card>
-          <img src="../assets/images/humm90Card.png" alt="Humm90 card" />
+        <Card v-for="card in cards.slice(0, 1)" :key="card.id" size="sm">
```

Comment:
I've hard-coded the size of the `cards` array and used splice here to follow a general pattern followed in other themes with more than 1 card.

Happy to change for a better implementation

Reply:
Can you move the data in `v-for` to a computed function? So we don't have unneeded logic in the template